### PR TITLE
Migrating urlresolver topic to queries subdir as per Magedoc3977

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) moves the urlResolver endpoint to the new queries subdir.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/reference/url-resolver.html

### CODE REVIEW

File: guides/v2.3/graphql/queries/url-resolver.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  "data": {
+    "urlResolver": {
+      "id": 1,
+      "relative_url": "catalog/product/view/id/1",
+      "type": "PRODUCT"
+    }
+  }
+}
+```
+
```
Comment (keharper) [inline guides/v2.3/graphql/queries/url-resolver.md]: `url` is an input parameter.

File: guides/v2.3/graphql/queries/url-resolver.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Attribute |  Data Type | Description
+--- | --- | ---
+`canonical_url` | String | Deprecated. Use `relative_url` instead.
+`id` | Int | The ID assigned to the object associated with the specified `url`. This could be a product ID, category ID, or page ID.
+`relative_url` | String | The internal relative URL. If the specified  `url` is a redirect, the query returns the redirected URL, not the original.
+`type` | UrlRewriteEntityTypeEnum | The value of `UrlRewriteEntityTypeEnum` is one of PRODUCT, CATEGORY, or CMS_PAGE.
+`url` | String | The URL to resolve. Magento stores product and category URLs with the `.html` extension.  CMS URLs do not contain the extension.
+
+### HttpQueryParameter object {#HttpQueryParameter}
```
Comment (keharper) [inline guides/v2.3/graphql/queries/url-resolver.md]: Move this section to the product query, beneath UrlRewrite object

File: guides/v2.3/graphql/reference/products.md [hunk 3]
```diff
@@ -361,14 +353,34 @@ Attribute | Type | Description
...
 ### SortField object
 
 Attribute | Type | Description
 --- | --- | ---
 `label` | String | The attribute's label
 `value` | String | The attribute name or code to use as the sort field
 
+### Website object {#websiteObject}
+
+Use the `Website` attributes to retrieve information about the website's configuration; such as, website name, website code, and default group id.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/products.md]:
```suggestion
Use the `Website` attributes to retrieve information about the website's configuration, which includes the website name, website code, and default group ID.
```

File: guides/v2.3/graphql/queries/url-resolver.md (hunk 1 above)
Comment (erikmarr) [reply guides/v2.3/graphql/queries/url-resolver.md]: added new input attributes section

File: guides/v2.3/graphql/queries/url-resolver.md (hunk 2 above)
Comment (erikmarr) [reply guides/v2.3/graphql/queries/url-resolver.md]: moved

File: guides/v2.3/graphql/reference/products.md (hunk 3 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/products.md]: fixed

**Comment by erikmarr** [comment]
running tests

**Pull request merged**